packages/currency/src/types.ts

```diff
@@ -1,13 +1,33 @@
-import { RequestLogicTypes } from '@requestnetwork/types';
+import { CurrencyTypes, RequestLogicTypes } from '@requestnetwork/types';
+
+/**
+ * Common types used in token configuration files
+ */
+type TokenAddress = string;
+type TokenDefinition = { name: string; symbol: string; decimals: number };
+export type TokenMap = Record<TokenAddress, TokenDefinition>;
+export type ChainDefinition = {
```

Comment:
Can you come up with another name for this type? It's not natural to "define" a chain by its currency map. Maybe `ChainDetails` or `ChainTokenMap`? Also, this type should be documented in the Currency README, it becomes key.

Reply:
fixed in a446daf0

Reply:
fixed in a446daf0